app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionController.scala

```diff
@@ -187,12 +189,16 @@ class SubscriptionController @Inject()
   val showSubscriptionComplete: Action[AnyContent] = AuthorisedWithSubscribingAgentAsync {
     implicit authContext =>
       implicit request => {
-        Future successful{
+        Future successful {
           val agencyData = for {
             agencyName <- request.flash.get("agencyName")
             arn <- request.flash.get("arn")
           } yield (agencyName, arn)
 
+          authenticatorConnector.refreshEnrolments.map { httpResponse =>
+            Logger.info(">>>>> AUTHENTICATOR RESPONSE: " + httpResponse.status)
```

Comment:
You could also put everything here in the same for yield statement ;) just use _ = for the logging thing